documentation/BrowserChannel.md

```diff
@@ -82,7 +82,7 @@ The workflow between the client and server can be summarised as follows:
 The client has been checked in to [../examples/browserChannelClient](../examples/browserChannelClient). To try it out, first
 start the GPII in the CloudBased browserChannel test configuration from the root of universal with
 
-    node gpii.js gpii/configs gpii.config.cloudBased.production
+    node gpii.js gpii/configs gpii.config.development.all.local
```

Comment:
Actually it will be more helpful to direct the user to one of our mock configurations like https://github.com/GPII/universal/blob/master/gpii/configs/mocks/gpii.config.development.all.local.mock.windows.json5 which will allow them to see the results of a wider range of preference sets